app/src/ui/branches/pull-requests-loading.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import { FilterList, IFilterListItem } from '../lib/filter-list'
+import { Octicon, OcticonSymbol } from '../octicons'
+
+const RowHeight = 45
+
+const FacadeCount = 6
+
+/**
+ * TS can't parse generic specialization in JSX, so we have to alias it here
+ * with the generic type. See https://github.com/Microsoft/TypeScript/issues/6395.
+ */
+const PullRequestsLoadingList: new () => FilterList<
+  IFilterListItem
+> = FilterList as any
+
+export class PullRequestsLoading extends React.Component<{}, {}> {
+  public render() {
+    const items: Array<IFilterListItem> = []
+    for (let i = 0; i < FacadeCount; i++) {
+      items.push({
+        text: '',
+        id: i.toString(),
+      })
+    }
+
+    const groups = [
+      {
+        identifier: '',
+        items,
+      },
+    ]
+
+    return (
+      <PullRequestsLoadingList
+        className="pull-request-list"
+        rowHeight={RowHeight}
+        groups={groups}
+        selectedItem={null}
+        renderItem={this.renderItem}
+        invalidationProps={groups}
+        filterDisabled={true}
+      />
+    )
+  }
+
+  private renderItem = (item: IFilterListItem) => {
+    return (
+      <div className="pull-request-loading-item">
```

Comment:
How about a `style={{ opacity: (FacadeCount - item.id)/FacadeCount }}` to make them fade out towards the bottom?

Reply:
Perhaps! @niik already has in mind a wave/pulse animation for these.